requirements-dev.txt

```diff
@@ -14,3 +14,5 @@ lintrunner-adapters>=0.8
 # Dependencies for the reference implementation.
 google-re2
```

Comment:
Can you delete this line now that you have added the line `-r requirements-reference.txt`

Reply:
Done